File: desktop-client/ECommerceAdminClient/Models/OrderItemDTO.cs
using System.Text.Json.Serialization;

namespace ECommerceAdminClient.Models
{
    public class OrderItemDTO
    {
        [JsonPropertyName("productName")]
        public string ProductName { get; set; }

        [JsonPropertyName("quantity")]
        public int Quantity { get; set; }

        [JsonPropertyName("purchasePrice")]
        public double PurchasePrice { get; set; }
    }
}

File: backend/src/main/java/com/example/E_CommerceSCD/DTOs/OrderItemDTO.java
package com.example.E_CommerceSCD.DTOs;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class OrderItemDTO {
    private String productName;
    private int quantity;
    private double purchasePrice;
}